SU2_CFD/src/solvers/CFlameletSolver.cpp

```diff
@@ -535,31 +535,34 @@ void CFlameletSolver::Source_Residual(CGeometry *geometry,
   CFluidModel *fluid_model_local = solver_container[FLOW_SOL]->GetFluidModel();
   
   SU2_OMP_FOR_DYN(omp_chunk_size)
-  for (auto i_point = 0u; i_point < nPointDomain; i_point++) {
+  for (auto iPoint = 0u; iPoint < nPointDomain; iPoint++) {
```

Comment:
please separate cleanup commits like these parts from actual code changes. And separate them logically as well to some degree, like the strongBC thing would be its own commit. This make looking through the commits much easier than this big chunk. Doesn't need to be perfect but some care makes the overview much easier as there is some people's work depending on feature_flamelet.